app/workers/evss/document_upload.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-class EVSSClaim::DocumentUpload
+class EVSS::DocumentUpload
```

Comment:
There will probably be workers stuck in the queue with this class (failed to upload and are being retried), you could either update those workers with a rake task/migration to use this new class, or create a new class with the old name and pass through all the args to the new class. The other class is less likely to have failures but probably a good idea to follow the same migration path